shell/packages/sandstorm-db/db.js

```diff
@@ -582,6 +582,12 @@ matchWildcardHost = function(host) {
 
   var prefix = wildcardHost[0];
   var suffix = wildcardHost[1];
+
+  // We ignore everything after the first : character to be agnostic
```

Comment:
Hmm. Technically, someone out there could be running sandstorm on `example.com:6080` and using it to web-publish to `example.com`, and I think this would break them?